Source/Krypton Components/Krypton.Toolkit/ButtonSpec/ButtonSpecView.cs

```diff
@@ -379,21 +379,19 @@ private void OnClick(object sender, MouseEventArgs e)
             // Never show a context menu in design mode
             if (!CommonHelper.DesignMode(Manager.Control))
             {
-                var showMenu = false;
-                var performDefaultClick = true;
-                if (ButtonSpec is ButtonSpecAny { ShowDrop: true })
-                {
-                    showMenu = ViewButton?.SplitRectangle.Contains(e.Location) ?? false;
-                    performDefaultClick = !showMenu;
-                }
+                // ButtonSpec's used to drop menu's if they had a context menu;
+                // BUT; Disable default action, if this is a drop button and it is clicked
+                bool performDefaultClick = !(ButtonSpec is ButtonSpecAny { ShowDrop: true }
+                                             && ViewButton != null
+                                             && ViewButton.SplitRectangle.Contains(e.Location));
 
                 if (performDefaultClick)
                 {
                     // Fire the event handlers hooked into the button spec click event
                     ButtonSpec.PerformClick(e);
                 }
 
-                if (showMenu)
+                // ToDo: Is this 'else' or remove brackets?
```

Comment:
Remove brackets, so that it performs as per V70

Reply:
Removed brackets